packages/fx-core/src/component/generator/utils.ts

```diff
@@ -117,6 +117,23 @@ export async function fetchTemplateZipUrl(
   if (!selectedTag) {
     throw new Error(`Failed to find valid template for ${name}`);
   }
+  return selectedTag;
+}
+
+export function getTemplateLocalVersion(): string {
+  return templateConfig.localVersion;
+}
+
+export async function fetchTemplateZipUrl(
```

Comment:
Is this function unused? Let's remove it if so.

Reply:
Yes, this function is only used in the ut. I will remove this function and modify the ut.